document-store/src/integrationTest/java/org/hypertrace/core/documentstore/mongo/MongoDocStoreTest.java

```diff
@@ -781,41 +781,81 @@ public void testLike() {
   }
 
   @Test
-  public void testNotEquals() {
-    MongoClient mongoClient = MongoClients.create("mongodb://localhost:27017");
+  public void testNotEquals() throws IOException {
```

Comment:
Previously this was testing behaviour of mongo client instead of Collection class